cognitedata/reveal
# feat(react-components): select single model simple list when has no scene

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:
[BND3D-3961](https://cognitedata.atlassian.net/browse/BND3D-3961)

https://cognitedata.atlassian.net/browse/

## Description :pencil:
This feature is to give the option to select single 3d models CAD  or PCs when doesnt have Scenes in the project. 
It will shows up a globe icon on the left toolbar when has no scene created for the project and when is clicked, it will open a list of 3d models to select. 

## How has this been tested? :mag:

Tested with simulating the project with no scenes by forcing the flag "hasScene" being false (returned by the hook `useCheckIfHasScene`)  in order to force the icon being show and list the 3d models available on the project to be selected, as no project without any scenes it was able to find and test it.


## Test instructions :information_source:

1. Go to the project with no scenes (or force false the flag returned by `useCheckIfHasScene `hook)
2. Open the Browser 3D
3. Select the globe icon on the left toolbar above the Layers icon
4. Select a 3d model
5. Observe the 3d model showing in the 3d viewer

Note: this PR comes along with the PR in Fusion: https://github.com/cognitedata/fusion/pull/8080



[BND3D-3961]: https://cognitedata.atlassian.net/browse/BND3D-3961?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
